src/tirith/providers/sg_workflow/handler.py

```diff
@@ -70,7 +71,7 @@ def __getValue(key, data):
     return result
 
 
-def provide(provider_args, input_data):
+def provide(provider_args: Dict[str, Any], input_data: Dict[st, Any]) -> List[Dict[str, Any]]:
```

Comment:
`Dict[st` . There's a typo here the `st`

Reply:
I fix the typo